api/client/trust.go

```diff
@@ -37,7 +37,12 @@ import (
 	"github.com/docker/notary/tuf/data"
 )
 
-var untrusted bool
+const releases = "releases"
+
+var (
+	releasesRole = filepath.Join(data.CanonicalTargetsRole, releases)
```

Comment:
Is that actually a path, or are we using `filepath.Join` as a mean to create a `"X/Y"` string? If it's the first I'm all for it, if not then we should make it explicit, because this will produce different strings on different OS.